config/prod.exs

```diff
@@ -20,7 +20,7 @@ config :logger, level: :info
 config :quadquizaminos, QuadquizaminosWeb.Endpoint, server: true
 
 config :quadquizaminos,
-  qna_directory: "/home/app/prod/lib/quadquizaminos/priv/static/qna"
+  qna_directory: "/home/app/prod/rel/quadquizaminos/qna"
```

Comment:
When making a release, it will throw an error `file not found`  trying to access the qna folder since the path configured here will not match. 

We need to put into consideration the path for qna when making release.

Reply:
Using the make command or just plain `MIX_ENV=prod mix release`?

Reply:
I ran `MIX_ENV=prod make release` and got the following result
```shell
mkdir -p priv/static
MIX_ENV=prod mix phx.digest
Check your digested files at "priv/static"
MIX_ENV=prod mix release
* assembling quadquizaminos-0.3.6 on MIX_ENV=prod
* using config/releases.exs to configure the release at runtime
Copying qna folder.....

Release created at _build/prod/rel/quadquizaminos!

    # To start your system
    _build/prod/rel/quadquizaminos/bin/quadquizaminos start

Once the release is running:

    # To connect to it remotely
    _build/prod/rel/quadquizaminos/bin/quadquizaminos remote

    # To stop it gracefully (you may also send SIGINT/SIGTERM)
    _build/prod/rel/quadquizaminos/bin/quadquizaminos stop

To list all commands:

    _build/prod/rel/quadquizaminos/bin/quadquizaminos
```

Reply:
Internal error is thrown when trying to access the Tetris page after starting the server.